README.md

```diff
@@ -16,15 +16,18 @@ from DataDistillr for use in machine learning.
 * `get_dict_from_api(url, auth_token, filename)`:  Pulls your data and returns it in a Python dictionary.
 
 #### DatadistillrAccount
-* `login(email, password)`: Logs you in to DataDistillr account.
 * `logout()`:  Logs you out of DataDistillr account.
 * `get_projects()`:  Returns all projects in DataDistillr account as a list of Project objects.
-* `get_project(project_name)`:  Finds project with project_name and returns Project object.
-* `get_organizations()`:  Returns list organizations that you have access.
+* `get_project_token_dict()`: Returns dictionary with project tokens as keys and project names as values.
+* `get_project(project_token)`:  Returns project object identified by project_token.
+* `get_organizations()`:  Returns list organizations that DataDistillr account has access to.
 
 #### Project
-* `get_tab(tab_name)`: Gets query results of given tab in project.
-* `query(tab_name, sql_statement)`:  Runs query and returns result of query.
+* `get_query_barrel_token_dict()`: Returns dictionary with query barrel tokens as keys and query barrel names as values. A query barrel is equivalent to a tab in the DataDistillr user interface.
```

Comment:
Somewhere, we should explain what the `queryBarrel` is....

Reply:
One more silly thing... The `Query Barrel` is the internal quasi-funny way we refer to the tabs.  I'm wondering if we should rename the functions in the SDK so that it would make sense to a user.   What do you think?

Other than that... LGTM +1

Reply:
ok I will change it to getTab(). This will be part of the next PR because its Project.py related